x/evm/precompile/funtoken.go

```diff
@@ -32,29 +30,23 @@ func (p precompileFunToken) Address() gethcommon.Address {
 	return PrecompileAddr_FunToken
 }
 
-func (p precompileFunToken) ABI() *gethabi.ABI {
-	return embeds.SmartContract_FunToken.ABI
-}
-
 // RequiredGas calculates the cost of calling the precompile in gas units.
 func (p precompileFunToken) RequiredGas(input []byte) (gasCost uint64) {
-	return RequiredGas(input, p.ABI())
```

Comment:
Isn't it easier to keep the ABI consistent when you only have to set its value in one place for the entire precompile?